src/alsa.c

```diff
@@ -1260,6 +1262,8 @@ static void instream_thread_run(void *arg) {
             case SND_PCM_STATE_DISCONNECTED:
                 instream->error_callback(instream, SoundIoErrorStreaming);
                 return;
+            case SND_PCM_STATE_PRIVATE1:
```

Comment:
Can you make this the default case so that it works with older alsa versions as well?